test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTestBase.java

```diff
@@ -166,6 +166,9 @@ public void run() throws Throwable
             if (nodesToUpgrade.isEmpty())
                 for (int n = 1; n <= nodeCount; n++)
                     nodesToUpgrade.add(n);
+                
+            if (configConsumer == null)
+                configConsumer = config -> config.set("auto_bootstrap", true);
```

Comment:
@ekaterinadimitrova2 @adelapena This is wild. I'll have more commentary in CASSANDRA-16387 soon, but I don't know how it's ever been safe for `AbstractCluster#startup()` to start multiple nodes concurrently. 3.0 nodes trying to create the traces keyspace concurrently seem to be able message each other with schema changes before the other creates the local entry in the `Gossiper` endpoint state map, and you get ugly assertions.

Reply:
CC @ifesdjeen